install.sh

```diff
@@ -356,11 +356,9 @@ parse_flags ()
             --with-qt)
                 with_qt='1'
                 ;;
-            -?*)
-                echo "warning.  unknown flag : $1" 1>&2
-                ;;
             *)
-                break
+                echo -e "\nUsage: "${0}" [options]\n\nOptions:\n\n--develop       code remains editable in place\n--python, -p    python version (default: python2)\n--with-qt       build the Qt GUI (incompatible with python2)\n"
```

Comment:
No strong feelings about this, but if you preformat the thing printed by `echo` like
```
echo "
    Usage ...
    option1, info option1
    option2, info option2
    ...
```
Then adding or removing something will be an easier diff to look at instead of the whole line changing

Reply:
I agree with this.